src/CMakeLists.txt

```diff
@@ -108,6 +108,7 @@ if(ENABLE_SWIFT)
                     TARGET
                       ${CMAKE_C_COMPILER_TARGET}
                     CFLAGS
+                      -D__DISPATCH_BUILDING_SWIFT_DISPATCH__
```

Comment:
Please don't add this.  You already have `dispatch_EXPORTS` provided for you by CMake.

Reply:
Do you mean DISPATCH_EXPORT? dispatch_EXPORTS doesn't appear anywhere in the libdispatch project.

Reply:
`dispatch_EXPORTS` is defined implicitly by CMake.